app/Resources/translations/default/ModulesGAnalyticsAdmin.xlf

```diff
@@ -66,9 +66,9 @@ File: modules/ps_googleanalytics/views/templates/admin/configuration.tpl:47</not
         <note>Context:
 File: modules/ps_googleanalytics/views/templates/admin/configuration.tpl:48</note>
       </trans-unit>
-      <trans-unit id="ade6a18bb6c147db87bc9463240e455a">
-        <source>"Add to cart" actions</source>
-        <target>"Add to cart" actions</target>
+      <trans-unit id="c2083746978ad21222cccce025233610">
+        <source>Add to cart" actions</source>
+        <target>Add to cart" actions</target>
```

Comment:
Thos translation seems to be wrong

https://github.com/PrestaShop/ps_googleanalytics/blob/4258d1c42d3aadade55cfa830b77c3291b045ec3/views/templates/admin/configuration.tpl#L49